.changeset/tricky-cobras-fold.md

```diff
@@ -0,0 +1,11 @@
+---
+'@talend/scripts-config-storybook-lib': minor
+'@talend/design-system': minor
+'@talend/react-cmf-router': minor
+'@talend/react-containers': minor
+'@talend/react-cmf': minor
+'@talend/scripts-publish-local': patch
+'@talend/scripts-locales': patch
+---
```

Comment:
For packages with major dependency upgrade, I use a minor changeset.
For packages with patch dependency upgrade, I use a patch changeset.